projects/go-lib/src/lib/components/go-toast/go-toast.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <div
     class="go-toast__container"
-    [ngClass]=" enableMaxHeight?'go-toast__container--max-height':'go-toast__container' ">
+    [ngClass]="enableMaxHeight?'go-toast__container--max-height':'go-toast__container'">
```

Comment:
I apologize for yet another request, but please add spaces around the question marks and colons. If you copy paste the line from my last comment it should be good.

Reply:
Actually, here you go. Just copy paste this:
```
[ngClass]="enableMaxHeight ? 'go-toast__container--max-height' : 'go-toast__container'"
```